1.24.0/rockcraft.yaml

```diff
@@ -0,0 +1,53 @@
+# Rock for [install-cni](https://github.com/istio/istio/blob/master/cni/deployments/kubernetes/Dockerfile.install-cni).
+# This image's purpose is described [here](https://github.com/istio/istio/tree/master/cni#overview)
+
+name: install-cni
+summary: Istio's install-cni in a rock
+description: "Installs Istio's cni plugin"
+version: "1.24.0"
```

Comment:
I think we need 1.23.3 as well,

Reply:
bumped this to 1.24.1.  Will add 1.23.3 as a separate PR

Reply:
1.23.3 is in #4